src/software/Localization/convertDescFloat2Uchar.cpp

```diff
@@ -2,10 +2,13 @@
 #include <openMVG/logger.hpp>
 
 #include <boost/progress.hpp>
-#include <boost/filesystem.hpp>
 #include <boost/program_options.hpp> 
 #include <boost/algorithm/string/case_conv.hpp> 
 
+#define BOOST_NO_CXX11_SCOPED_ENUMS
```

Comment:
@fabiencastan Oh, I saw the option, but didn't know what it means. The problem is anyway that I'm using system boost (on Debian) and it's not possible to tell upfront how it's compiled.